apps/connect/source/ftrack_connect/util.py

```diff
@@ -179,3 +184,111 @@ def wrapper(*args, **kwargs):
         return invoke_in_qt_main_thread(func, *args, **kwargs)
 
     return wrapper
+
+
+def get_plugin_json_url_from_environment():
+    '''Return plugin json url from environment variable'''
+    return os.environ.get('FTRACK_CONNECT_JSON_PLUGINS_URL')
+
+
+def check_major_version(version, major_version_start=24):
+    match = re.match(r'v(\d+)\.\d+\.\d+.*', version)
+    if match:
+        major_version = int(match.group(1))
+        return major_version >= major_version_start
+    else:
+        return False
+
+
+def fetch_github_releases(latest=True):
```

Comment:
Shouldn't you add a check to make sure to filter out pre-releases? Also, should we be able to change the repository with an environment variable as we do for the beta channel?

Reply:
Yes, pre-releases should be ignored.

We could add an environment for that, but it might be to their private repo and so we would need to also supply credentials. Let's discuss